### PR TITLE
Add Dependency Injection Container extension. Add CS fixer recipe and apply CS fixing

### PR DESCRIPTION
**Pull request by PedroTroller**

### CODE REVIEW

File: src/Knp/Rad/Security/EventListener/AuthorizationListener.php [hunk 1]
```diff
@@ -25,10 +25,10 @@ public function checkIfUserIsGranted(FilterControllerEvent $event)
     {
         $request = $event->getRequest();
 
-        foreach ($request->attributes->get('_security', array()) AS $rule) {
+        foreach ($request->attributes->get('_security', array()) as $rule) {
             $roles = array();
 
-            if (isset($rule['roles']) && !empty($rule['roles'])) {
+            if (isset($rule['roles']) && ! empty($rule['roles'])) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/EventListener/AuthorizationListener.php:31]: Why do you add an extra space here ?

File: src/Knp/Rad/Security/EventListener/AuthorizationListener.php [hunk 2]
```diff
@@ -41,8 +41,8 @@ public function checkIfUserIsGranted(FilterControllerEvent $event)
             $subject     = null;
             $subjectName = isset($rule['subject']) ? $rule['subject'] : null;
 
-            if (!empty($subjectName)) {
-                if (!$request->attributes->has($subjectName)) {
+            if ( ! empty($subjectName)) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/EventListener/AuthorizationListener.php:44]: Same here.

File: src/Knp/Rad/Security/EventListener/AuthorizationListener.php [hunk 3]
```diff
@@ -41,8 +41,8 @@ public function checkIfUserIsGranted(FilterControllerEvent $event)
...
+                if ( ! $request->attributes->has($subjectName)) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/EventListener/AuthorizationListener.php:45]: Same here.

File: src/Knp/Rad/Security/EventListener/AuthorizationListener.php [hunk 4]
```diff
@@ -51,7 +51,7 @@ public function checkIfUserIsGranted(FilterControllerEvent $event)
                 $subject = $request->attributes->get($subjectName);
             }
 
-            if (!$this->checker->isGranted($roles, $subject)) {
+            if ( ! $this->checker->isGranted($roles, $subject)) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/EventListener/AuthorizationListener.php:54]: Same here.

File: src/Knp/Rad/Security/Voter/IsOwnerVoter.php [hunk 5]
```diff
@@ -32,15 +32,15 @@ public function supportsClass($class)
     public function vote(TokenInterface $token, $object, array $attributes)
     {
         foreach ($attributes as $attribute) {
-            if (!$this->supportsAttribute($attribute)) {
+            if ( ! $this->supportsAttribute($attribute)) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/Voter/IsOwnerVoter.php:35]: Same here.

File: src/Knp/Rad/Security/Voter/IsOwnerVoter.php [hunk 6]
```diff
@@ -32,15 +32,15 @@ public function supportsClass($class)
...
                 continue;
             }
 
-            if (!$this->supportsClass($object)) {
+            if ( ! $this->supportsClass($object)) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/Voter/IsOwnerVoter.php:39]: Same here.

File: src/Knp/Rad/Security/Voter/IsOwnerVoter.php [hunk 7]
```diff
@@ -32,15 +32,15 @@ public function supportsClass($class)
...
                 return self::ACCESS_ABSTAIN;
             }
 
-            if (!$token->getUser() instanceof OwnerInterface) {
+            if ( ! $token->getUser() instanceof OwnerInterface) {
```
Comment (akerouanton) [inline src/Knp/Rad/Security/Voter/IsOwnerVoter.php:43]: Same here.

File: src/Knp/Rad/Security/EventListener/AuthorizationListener.php (hunk 1 above)
Comment (einenlum) [inline src/Knp/Rad/Security/EventListener/AuthorizationListener.php:31]: @NiR-  https://github.com/KnpLabs/rad-security/pull/11/files#diff-b31b99fa4c2c185ba6a71235efb48224R15

**Pull request merged**